src/gpodder/gtkui/shownotes.py

```diff
@@ -201,7 +201,14 @@ def update(self, heading, subheading, episode):
         else:
             self._base_uri = episode.channel.url
         self._loaded = False
-        self.html_view.load_html(episode.description_html, self._base_uri)
+
+        description = episode.description_html
+        if not description:
+            description = self.coerce_text_to_html(episode.description)
+        self.html_view.load_html(description, self._base_uri)
```

Comment:
Then I can simply use self.html_view.load_plain_text, removing the need to `coerce_text_to_html`.